Dockerfile-build

```diff
@@ -0,0 +1,30 @@
+FROM mhart/alpine-node:4.8.0
+
+RUN apk update && apk upgrade && apk add git make gcc g++ linux-headers openssl python
+
+RUN addgroup -g 10001 app && \
+    adduser -D -G app -h /app -u 10001 app
+WORKDIR /app
+
+USER app
+
+COPY npm-shrinkwrap.json npm-shrinkwrap.json
+COPY package.json package.json
+COPY scripts/download_l10n.sh scripts/download_l10n.sh
+COPY scripts/gen_keys.js scripts/gen_keys.js
+COPY scripts/gen_vapid_keys.js scripts/gen_vapid_keys.js
+
+RUN npm install --production && rm -rf ~app/.npm /tmp/*
+
+COPY . /app
+USER root
+RUN chown app:app /app/config
+
+USER app
+RUN node scripts/gen_keys.js
+RUN node scripts/gen_vapid_keys.js
+
+USER root
+RUN chown root:root /app/config
```

Comment:
I don't think you need to worry about changing the permissions here - we'll just ignore the config file